src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -2014,12 +2019,26 @@ public SBApiResponse profileMDOAdminUpdate(Map<String, Object> request, String u
 					existingProfileDetails.put(Constants.PROFILE_GROUP_STATUS, Constants.VERIFIED);
 					existingProfileDetails.put(Constants.PROFILE_DESIGNATION_STATUS, Constants.VERIFIED);
 					isGroupOrDesignationUpdated = true;
-				} else if (Constants.NOT_VERIFIED.equalsIgnoreCase(updatedProfileStatus)
-						|| Constants.NOT_MY_USER.equalsIgnoreCase(updatedProfileStatus)) {
+				} else if (Constants.NOT_VERIFIED.equalsIgnoreCase(updatedProfileStatus)) {
 					//If marked as "NOT-VERIFIED" then no need to change the details.
 					isGroupOrDesignationUpdated = false;
```

Comment:
Just add -> existingProfileDetails.remove(Constants.UPDATE_AS_NOT_MY_USER);

in  the above if condition too... so that if this field exists it will be removed